docs/developer-docs/gas-cost.md

```diff
@@ -109,47 +111,45 @@ The amount of reserved cycles depends on how many bytes are allocated and on the
 - If subnet usage is above `450GiB`, then the amount of reserved cycles per allocated byte grows linearly depending on the subnet usage, from `0` to `10` years worth of storage payments at the subnet capacity (which is currently `750GiB`).
 
 A controller of a canister can disable resource reservation by setting the `reserved_cycles_limit=0` in canister settings.
-Such opted-out canisters would not be able to allocate if the subnet usage is above `450GiB` though.
+Such opted-out canisters would not be able to allocate if the subnet usage is above `450GiB`.
 
 ### Special features
 
-Special features aside from messaging, storage, and execution resources have different costs since they use special infrastructure and architectures to provide the feature's functionality. These special features include:
+Special features have different costs since they use special infrastructure to provide the feature's functionality. These special features include:
 
-- **HTTPS outcalls**: The cost of HTTPS outcalls is calculated using a unique formula since the feature includes a quadratic component in its implementation. The cost for an HTTPS outcall is calculated using the formula `(3_000_000 + 60_000 * n) * n` for the base fee and `400 * n` each request byte and `800 * n` for each response byte, where `n` is the number of nodes in the subnet. These costs are included in the chart found below.
+- **HTTPS outcalls**: The cost for an HTTPS outcall is calculated using the formula `(3_000_000 + 60_000 * n) * n` for the base fee and `400 * n` each request byte and `800 * n` for each response byte, where `n` is the number of nodes in the subnet. These costs are included in the chart found below.
 
 - **Bitcoin API**: Pricing for the **Bitcoin API** is available in the [Bitcoin API documentation](./integrations/bitcoin/bitcoin-how-it-works.md).
 
 - **Chain-key signing API**: Pricing for the **Chain-key signing API** is available in the [Chain-key signing / threshold ECDSA documentation](./integrations/t-ecdsa/t-ecdsa-how-it-works.md).
 
 ### Replication
 
-Replication refers to the number of times a canister is replicated, which will depend on the subnet that the canister is deployed to.
+Replication refers to the number of times a canister is replicated, which is dependant on the canister's subnet.
 
 #### Local development
 
-When a canister is deployed to a local development environment, the canister is deployed to a single node (the locally running instance of the replica). Cycles charged to locally deployed canisters have a cost that is 1/13th the cost when deployed to a 13-node subnet. In local development environments, cycles can be fabricated using dfx.
+When a canister is deployed to a local development environment, the canister is deployed to a single node. Cycles charged to locally deployed canisters have a cost that is 1/13th the cost when deployed to a 13-node subnet. In local development environments, cycles can be fabricated using dfx.
```

Comment:
Makes me think about possible improvements in our tooling.
* It seems very complicated to have a 1/13th of the cost locally. Can we not just completely remove any explanation of the local setup? Do I need to care about this as a developer? 
* Could we implement more of a prediction where the dev could be told how much he would need to pay if they were to run what they have locally on an n-node subnet on the IC.

Reply:
I think it's important to keep information on the local setup, since cycles are still charged in local development. This has been a point that the DevRel team has made efforts to emphasis, so I think it should be kept in to not give the impression that cycles aren't charged locally.